cluster-operator/src/test/java/io/strimzi/operator/cluster/model/JvmOptionsTest.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright 2018, Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.operator.cluster.model;
+
+import io.strimzi.operator.cluster.ResourceUtils;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertTrue;
+
+public class JvmOptionsTest {
+    @Test
+    public void testXmxXms() {
+        JvmOptions opts = JvmOptions.fromJson("{\n" +
+                "  \"-Xmx\": \"2g\",\n" +
+                "  \"-Xms\": \"1g\"\n" +
+                "}");
+
+        assertEquals("1g", opts.getXms());
+        assertEquals("2g", opts.getXmx());
+    }
+
+    @Test
+    public void testEmptyXmxXms() {
+        JvmOptions opts = JvmOptions.fromJson("{}");
+
+        assertNull(opts.getXms());
+        assertNull(opts.getXmx());
+    }
+
+    @Test
+    public void testServer() {
+        JvmOptions opts = JvmOptions.fromJson("{\n" +
+                "  \"-server\": \"true\"\n" +
+                "}");
+
+        assertTrue(opts.getServer());
+
+
+        opts = JvmOptions.fromJson("{\n" +
+                "  \"-server\": true\n" +
+                "}");
+
+        assertTrue(opts.getServer());
+
+        opts = JvmOptions.fromJson("{\n" +
+                "  \"-server\": \"false\"\n" +
+                "}");
```

Comment:
one line for this simple JSON ? The same for the others above ?

Reply:
Might be subjective, but for me it is much easier to read and edit like this. The only place where I used one line JSON was when adding something to existing one line JSONs.